django_filters/filters.py

```diff
@@ -484,6 +486,38 @@ class TimeRangeFilter(RangeFilter):
     field_class = TimeRangeField
 
 
+class SearchVectorFilter(Filter):
```

Comment:
I would inherit `CharFilter`, as you are validating text.

Reply:
Reasonably and thus accepted.